Docker.DotNet/Properties/AssemblyInfo.cs

```diff
@@ -29,5 +29,5 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.16.0.0")]
+[assembly: AssemblyFileVersion("1.16.0.0")]
```

Comment:
@jgarverick ok I guess I'll just merge like this and then take a fix. Thanks. I hope all goes well for your daughter. :heart: